transform/utils/utils.py

```diff
@@ -108,3 +110,17 @@ def extract_digits_and_trim(input_str: str):
     remaining_str = input_str[index:].strip()
 
     return remaining_str
+
+
+def map_currency_symbol(currency_symbol):
+    """Maps a currency symbol to its corresponding value."""
+    return currency_mapping.get(currency_symbol, currency_symbol)
+
+
+def title_case_funct(s):
```

Comment:
- if it will be used only in projects - please define it as a method in Project class 
- Could you elaborate a little bit more in the doc string on what this function is actually doing? Its not that clear for me :D

Reply:
deleted